client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionEventForm/useInteractionsForm.ts

```diff
@@ -15,114 +15,117 @@ import useStyles from './InteractionEventFormStyles';
 import {duplicateIdsErrorMsg} from '../../ContactQuestioning/ContactQuestioning';
 
 const useInteractionsForm = (props: useInteractionFormIncome): useInteractionFormOutcome => {
-    const classes = useStyles();
-    const swalClasses = useSwalStyles();
-    const {loadInteractions, closeNewDialog, closeEditDialog} = props;
-    const {parseLocation} = useDBParser();
-    const epidemiologyNumber = useSelector<StoreStateType, number>(state => state.investigation.epidemiologyNumber);
-    const userId = useSelector<StoreStateType, string>(state => state.user.id);
+        const classes = useStyles();
+        const swalClasses = useSwalStyles();
+        const {loadInteractions, closeNewDialog, closeEditDialog} = props;
+        const {parseLocation} = useDBParser();
+        const epidemiologyNumber = useSelector<StoreStateType, number>(state => state.investigation.epidemiologyNumber);
+        const userId = useSelector<StoreStateType, string>(state => state.user.id);
 
-    const checkForDuplicateIdsInContactEvent = (currContacts: Contact[]) => {
-        const allIdentificationNumbers = currContacts.map((contact) => contact.idNumber);
-        return allIdentificationNumbers.some((idNumber, idNumberIndex) => {
-            return allIdentificationNumbers.indexOf(idNumber) !== idNumberIndex && idNumber !== undefined
-        });
-    }
+        const checkForDuplicateIdsInContactEvent = (currContacts: Contact[]) => {
+            const allIdentificationNumbers = currContacts.map((contact) => contact.idNumber);
+            return allIdentificationNumbers.some((idNumber, idNumberIndex) => {
+                return allIdentificationNumbers.indexOf(idNumber) !== idNumberIndex && idNumber !== undefined
```

Comment:
idNumber  !== undefined is to check if idNumber is empty?

Reply:
yes

Reply:
and if it is an empty string it is really undefined? or it is ''

Reply:
pretty sure it's undefined but ill check again

Reply:
yes it's undefined